From 237b7d97b208e89aace2c86ade82dcba86986e6f Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 13 Apr 2018 15:17:58 -0400
Subject: [PATCH 06/55] Add some *more* TODO tasks.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 TODO | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/TODO b/TODO
index 2f6cc19..a2cad71 100644
--- a/TODO
+++ b/TODO
@@ -18,11 +18,14 @@
   - including vendor path that means "this image"
   - including path that's like Fv() to embed images.
 - Make all build options be able to be set in 'git config --local shim.OPTION'
+  - Make the build dump those to stdout as well
 - make debuginfo paths configurable
 - make arch dependent names configurable
 - Make it easier to avoid CryptPem
 - Make an easy strip+implant tool for our embedded cert lists
 - Post process full path names out of __FILE__ / __BASE_FILE__ entries in
   the string table :/
+- Make build.log an artifact of building.
+- KEK for Mok.  (koike expressed an interest in working on this.)
 
 # vim:filetype=mail:tw=74
-- 
2.17.1

