From 119dead3aea191f27ee4ccc487ae642503a786e6 Mon Sep 17 00:00:00 2001
From: Paul Menzel <pmenzel@molgen.mpg.de>
Date: Wed, 23 May 2018 12:43:45 +0200
Subject: [PATCH 08/55] README: Remove superfluous *and*

---
 README | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/README b/README
index ec9403a..0703049 100644
--- a/README
+++ b/README
@@ -9,8 +9,8 @@ will relocate and execute the binary.
 shim will also install a protocol which permits the second-stage bootloader
 to perform similar binary validation. This protocol has a GUID as described
 in the shim.h header file and provides a single entry point. On 64-bit systems
-this entry point expects to be called with SysV ABI rather than MSABI, and
-so calls to it should not be wrapped.
+this entry point expects to be called with SysV ABI rather than MSABI, so calls
+to it should not be wrapped.
 
 On systems with a TPM chip enabled and supported by the system firmware,
 shim will extend various PCRs with the digests of the targets it is
-- 
2.17.1

