From 0fd3c7e8518d25f7f51e970bd77f230d0e750018 Mon Sep 17 00:00:00 2001
From: Gary Lin <glin@suse.com>
Date: Mon, 28 May 2018 18:03:39 +0800
Subject: [PATCH 12/55] httpboot: show the error message for the ChildHandle

Signed-off-by: Gary Lin <glin@suse.com>
---
 httpboot.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/httpboot.c b/httpboot.c
index 16dd662..3622e85 100644
--- a/httpboot.c
+++ b/httpboot.c
@@ -696,8 +696,10 @@ http_fetch (EFI_HANDLE image, EFI_HANDLE device,
 	/* Set the handle to NULL to request a new handle */
 	http_handle = NULL;
 	efi_status = service->CreateChild(service, &http_handle);
-	if (EFI_ERROR(efi_status))
+	if (EFI_ERROR(efi_status)) {
+		perror(L"Failed to create the ChildHandle\n");
 		return efi_status;
+	}
 
 	/* Get the http protocol */
 	efi_status = gBS->HandleProtocol(http_handle, &EFI_HTTP_PROTOCOL_GUID,
-- 
2.17.1

